sdk/python/kubeflow/training/api/training_client.py

```diff
@@ -805,21 +858,26 @@ def get_job_logs(
             follow: Whether to follow the log stream of the pod and print logs to StdOut.
             timeout: Optional, Kubernetes API server timeout in seconds
                 to execute the request.
+            verbose: Whether to get Kubernetes events for Job and corresponding pods.
 
         Returns:
             Dict[str, str]: A dictionary in which the keys are pod names and the
             values are the corresponding logs.
+            Dict[str, str]: A dictionary in which the keys are Job and pod names and the
+            values are list of the corresponding Kubernetes events with their timestamps. This
+            value is returned only if `verbose = True`.
+
 
         Raises:
             ValueError: Job replica type is invalid.
-            TimeoutError: Timeout to get Job pods.
-            RuntimeError: Failed to get Job pods.
+            TimeoutError: Timeout to get Job or Job's pods
```

Comment:
```suggestion
            TimeoutError: Timeout to get Job or Job's pods.
```
Apologies for the nit! 🙃

Reply:
Thanks for that! I would like to keep our docstrings clean 😄